src/Routes.tsx

```diff
@@ -3,22 +3,20 @@ import { Switch, Route } from 'react-router-dom'
 import Landing from './pages/Landing'
 import Login from './pages/Login'
 import Register from './pages/Register'
-import JobSeeker from './pages/JobSeeker'
-import EditProfile from './pages/EditProfile'
 import PWReset from './pages/PWReset'
-import Profile from './pages/Profile'
-import CompanyMatch from './pages/CompanyMatch'
+import ViewJobseeker from './pages/ViewJobseeker'
+import CompanyProfile from './pages/CompanyProfile'
+import JobseekerProfile from './pages/JobseekerProfile'
 
 const Routes = () => (
   <Switch>
     <Route exact path="/" component={Landing} />
     <Route exact path="/login" component={Login} />
     <Route exact path="/register" component={Register} />
-    <Route exact path="/jobseeker/match" component={JobSeeker} />
-    <Route exact path="/edit-jobseeker-profile" component={EditProfile} />
     <Route exact path="/reset-password" component={PWReset} />
-    <Route exact path="/jobseeker/profile" component={Profile} />
-    <Route exact path="/company/profile" component={CompanyMatch} />
+    <Route exact path="/jobseeker/view" component={ViewJobseeker} />
+    <Route exact path="/company/profile" component={CompanyProfile} />
+    <Route exact path="/jobseeker/profile" component={JobseekerProfile} />
```

Comment:
What's the difference between `/jobseeker/view` and `/jobseeker/profile`?

Reply:
/jobseeker/view is the page the employer sees when clicking "view" button on the match page:
![jobseekerView](https://user-images.githubusercontent.com/45182330/103915084-0678c400-510b-11eb-918a-7867010698e0.png)
/jobseeker/profile is the profile page: 
![jobseekerProfile](https://user-images.githubusercontent.com/45182330/103915181-2a3c0a00-510b-11eb-836d-2df44dc3fa29.png)

If needed, I can change the profile pages to company/homepage and jobseeker/homepage and the jobseeker/view to jobseeker/profile. :)) I think what matters is that the pages of jobseeker and employer that are similar to each other are also named in a similar way.

Reply:
Ok I see. I think since you go to this `view` from the `match` page, it's better to name like `match/view/applicant/:id` Later we can use this `:id` param to fetch the relevant info about the applicant that was clicked on

Reply:
Ok, done. 
I don't know why the checks failed in the last push.

Reply:
It's the 3rd-party github action that I installed to experiment 😄 I think you can click on it to see why it failed

Reply:
I was thinking that I haven't seen that before. Nice one. Now I found the problem and hopefully it passes the checks.

Reply:
🎉 It passed, you can merge now